packages/server/src/tests/utilities/TestConfiguration.ts

```diff
@@ -225,21 +224,28 @@ export default class TestConfiguration {
 
   // use a new id as the name to avoid name collisions
   async init(appName = newid()) {
-    if (!this.started) {
+    if (this.initialised) {
```

Comment:
We had to start doing a lot more in here because we needed to be able to `await` server startup. This is because we need to wait until Redis is ready, which is done during server startup. When we were using `ioredis-mock` this wasn't necessary because there was nothing really to wait for.